src/components/tiles/table/cell-formatter.tsx

```diff
@@ -2,18 +2,38 @@ import React from "react";
 import { CalculatedColumn, RowHeightArgs } from "react-data-grid";
 import { kCellHorizontalPadding, kDefaultColumnWidth, kRowHeight, TRow } from "./table-types";
 import { useNumberFormat } from "./use-number-format";
+import { gImageMap } from "../../../models/image-map";
 
 import './cell-formatter.scss';
 
 export const formatValue = (
-    formatter: (n: number | { valueOf(): number }) => string, value: any,
-    width?: number, row?: TRow, rowHeight?: (args: any) => number
+    formatter: (n: number | { valueOf(): number }) => string,
+    value: any,
+    lookupImage: (value: string) => string|undefined,
+    width?: number,
+    row?: TRow,
+    rowHeight?: (args: any) => number,
   ) => {
   if ((value == null) || (value === "")) return <span></span>;
   const num = Number(value);
   if (!isFinite(num)) {
     const cellWidth = (width || kDefaultColumnWidth) - kCellHorizontalPadding;
     const height = rowHeight && row ? rowHeight({ row }) : kRowHeight;
+    if (gImageMap.isImageUrl(value)) {
+      if (lookupImage) {
+        const url = lookupImage(value);
+        if (url) {
+          return (
+            <div className="image-cell" style={{ height, width: cellWidth}}>
+              <img src={url}></img>
+            </div>
+          );
+        }
+      } else {
+        console.log('lookupImage not defined');
```

Comment:
`lookupImage` is not an optional parameter, so I would expect it's always defined here. Was there a case when it wasn't?

Reply:
Yeah, this is just leftover code, will remove.